cli/src/commands/test.ts

```diff
@@ -67,11 +87,27 @@ const chaosUnit = {
             [q,r,s] = validNeighbours[Math.floor(Math.random() * validNeighbours.length)] || [0,0,0];
 
             await player.dispatch({ name: 'MOVE_MOBILE_UNIT', args: [unitKey, q, r, s] }).then(res => res.wait());
-            await sleep(Math.floor(Math.random() * 2000)); // jitter
+            await sleep(Math.floor(Math.random() * 1000)); // jitter
+            // place building
+            if (buildings?.length > 0){
+                let selectedBuilding = buildings[Math.floor(Math.random() * buildings.length)];
+                const targetTileID = CompoundKeyEncoder.encodeInt16(NodeSelectors.Tile, 0, q, r, s);
+                const world = await getWorld(ctx);
+                const buildingOnTargetTile = getBuildingOnTile(world.buildings, targetTileID);
+                // console.log(`target tile (${targetTileID}) building: ${buildingOnTargetTile}`);
+                if (buildingOnTargetTile === undefined){
```

Comment:
```suggestion
                if (typeof buildingOnTargetTile === 'undefined'){
```

javascript is dumb and allows you to do....

```
globalThis.undefined = "lol";
"lol" == undefined // true
```

... so it's generally best to avoid directly checking equality of undefined.